### ISSUE TITLE
a11y: State changes not announced

### ISSUE THREAD

**Issue by anezthes** [vaadin/vaadin-checkbox#169]
VoiceOver (Chrome/Safari) does not correctly announce state changes unless using:
1) a pointer device;
2) pressing Control-Option-Space.

Comparatively, native checkboxes work with spacebar when focused.

**Comment by anezthes** [comment]
I'm guessing role="presentation" is the culprit.

Couldn't we just do:
```
<vaadin-checkbox>
  <label>
    <input type="checkbox" name="consoles">
    <span>Wii</span>
  </label>
</vaadin-checkbox>
```
?

**Comment by anezthes** [comment]
See https://github.com/vaadin/vaadin-checkbox/issues/168#issuecomment-608292610.

**Comment by rolfsmeds** [comment]
Closing in favor of https://github.com/vaadin/vaadin-checkbox/issues/181
